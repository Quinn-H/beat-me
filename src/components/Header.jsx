import React from 'react'

export default React.createClass({
  render () {
    return (
      <header className="main-header">
        <h1>BEAT ME</h1>
      </header>
    )
  }
})
