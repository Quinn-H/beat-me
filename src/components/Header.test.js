/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import Header from './Header'

test('Header renders correctly', () => {
  const stop = function () {}
  const start = function () {}
  const isPlaying = false
  const tree = renderer.create(
    <Header />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
