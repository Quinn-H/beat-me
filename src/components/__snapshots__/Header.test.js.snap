exports[`test Header renders correctly 1`] = `
<header
  className="main-header">
  <h1>
    BEAT ME
  </h1>
</header>
`;
